x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_timeline/alert_event.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LineAnnotation } from '@elastic/charts';
+import { EuiIcon } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import moment from 'moment';
+import { EventSchema } from '@kbn/investigation-shared';
+
+export const AlertEvent = ({ event }: { event: EventSchema }) => {
+  return (
+    <LineAnnotation
+      id={event.id}
+      domainType="xDomain"
+      marker={
+        <>
+          <div
+            style={{
+              marginTop: -10,
+              backgroundColor: 'red',
+              borderRadius: 4,
+              padding: 4,
+              display: 'flex',
+              height: 20,
+              width: 20,
+            }}
+          >
+            <EuiIcon type="warning" size="s" color="white" />
```

Comment:
I think we wanted to keep "dot" icon with red color for alert and "dot" icon with black color for annotation events